packages/eslint-config-react/package.json

```diff
@@ -12,20 +12,17 @@
     "eslintconfig"
   ],
   "peerDependencies": {
-    "eslint": ">=7",
-    "eslint-plugin-import": ">=2.22.0",
-    "eslint-plugin-jsx-a11y": ">=6.4.0",
-    "eslint-plugin-react": ">=7.23",
     "typescript": ">=4.2"
   },
   "dependencies": {
-    "@trampoline/eslint-config": "^0.2.3"
-  },
-  "devDependencies": {
     "eslint": "^7.29.0",
     "eslint-plugin-import": "^2.23.4",
     "eslint-plugin-jsx-a11y": "^6.4.1",
     "eslint-plugin-react": "^7.24.0",
+    "@rushstack/eslint-patch": "^1.1.0",
+    "@trampoline/eslint-config": "^0.2.3"
```

Comment:
Since we apply the patch here, we probably don't even need the the plugins that are already in the base config. We should just keep the ones that we add in this specific package.